torchvision/datapoints/__init__.py

```diff
@@ -12,3 +12,25 @@
     import warnings
 
     warnings.warn(_BETA_TRANSFORMS_WARNING)
+
```

Comment:
Not going to block over this, but I would prefer keeping logic out of `__init__.py`.